CRM/Core/Controller.php

```diff
@@ -457,8 +457,8 @@ public function addPages(&$stateMachine, $action = CRM_Core_Action::NONE) {
       if ($options) {
         $$stateName->setOptions($options);
       }
-      if (property_exists($$stateName, 'urlPath') && isset($_GET[CRM_Core_Config::singleton()->userFrameworkURLVar])) {
-        $$stateName->urlPath = explode('/', $_GET[CRM_Core_Config::singleton()->userFrameworkURLVar]);
+      if (property_exists($$stateName, 'urlPath')) {
+        $$stateName->urlPath = explode('/', (string) CRM_Utils_System::getUrlPath());
```

Comment:
Well this property is something I added pretty recently and it's rarely used. I'm not worried about this cleanup breaking anything.
One of the few places I can think of where this variable is used is places like `/civicrm/admin/options/activity_type` where the form reads the last argument from the url. That's still working fine.